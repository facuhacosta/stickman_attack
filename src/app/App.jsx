import React from 'react';
import Style from './app.module.scss';
import { GameProvider } from './components/game-context/game-context';
import { Router } from './components/router';


function App() {

  return (
    <div className={Style.app}>
      <GameProvider>
        <Router/>
      </GameProvider>
    </div>
  );
};

export default App;