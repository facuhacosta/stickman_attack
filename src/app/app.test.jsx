import React from 'react';
import { render } from '@testing-library/react';
import App from './App';

describe('App Component', () => {

  // Matching Snapshots
  test('App should match the snapshot', () => {
    const { asFragment } = render(<App />);
    expect(asFragment()).toMatchSnapshot();
  });

});