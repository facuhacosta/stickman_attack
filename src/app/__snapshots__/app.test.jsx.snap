// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App Component App should match the snapshot 1`] = `
<DocumentFragment>
  <div>
    <div>
      <form>
        <input
          name="username"
          placeholder="Username"
          type="text"
          value=""
        />
        <input
          name="password"
          placeholder="asd123"
          type="password"
          value=""
        />
        <p />
        <div>
          <button
            class="undefined undefined"
            type="submit"
          >
            Log In
          </button>
        </div>
      </form>
    </div>
  </div>
</DocumentFragment>
`;
